File: Sem- 3/Data Structure/Array/Armstrong.java
public class Armstrong{
	public static void main(String[] args){
		System.out.println("Armstrong Numbers from 1 to 1000");
		for(int n=1;n<=1000;n++){
			int sum=0;
			int num=n;
			while(num!=0){
				int rem=num%10;
				sum+=(rem*rem*rem);
				num=num/10;
			}
			if(n==sum){
				System.out.print(n+" ");
			}
		}
	}
}

File: Sem-1/C/While Loop/armstrong.c
#include<stdio.h>
#include<math.h>
void main(){
	int n,i,count=0,sum=0,b,c;
	printf("Armstrong Number");
	printf("\nEach digit raised to the power of total number of digits when added equals to the number itself");
	printf("\nEnter any number: ");
	scanf("%d",&n);
	b=n;
	c=n;
	while(n!=0){
		i=n%10;
		n=n/10;
		count++;
	}
	while(b!=0){
		i=b%10;
		b=b/10;
		sum=sum+pow(i,count);
	}
	if(sum==c){
		printf("Armstrong Number");
	}
	else{
		printf("Not an Armstrong Number");
	}
}
